modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -264,7 +277,8 @@ struct IEUnit {
             this_request.SetBlob(p.first, wrapIE(p.second.first, p.second.second));
         }
 
-        return {this_plugin, this_network, this_request};
+        return std::shared_ptr<cv::gimpl::ie::IECompiled>(
+                new cv::gimpl::ie::IECompiled{this_plugin, this_network, this_request});
```

Comment:
`make_shared`?

Reply:
No, because `IECompiled` is a struct without `ctor`. 
C++11 can't manage it !!! :(